packages/oi4-oec-service-node/src/application/OI4ApplicationResources.ts

```diff
@@ -109,7 +109,9 @@ class OI4ApplicationResources extends ConfigParser implements IOI4ApplicationRes
 
     private static extractMamFile(path: string): MasterAssetModel {
         if (existsSync(path)) {
-            return JSON.parse(readFileSync(path).toString());
+            const mam = new MasterAssetModel();
```

Comment:
```suggestion
            return MasterAssetModel.clone(JSON.parse(readFileSync(path, 'utf8')));
```

There is already a clone function on the MasterAssetModel class, that is among others intended for JSON deserialization

Reply:
`clone` function is now used